.gitignore

```diff
@@ -0,0 +1,70 @@
+# Logs
```

Comment:
Can we get this file out of the PR. It's a little larger than most standard .gitignore files.